lib/management/user.ts

```diff
@@ -521,15 +521,24 @@ const withUser = (sdk: CoreSdk, managementKey?: string) => {
      * Note: The 'Manage tokens from provider' setting must be enabled.
      * @param loginId the login ID of the user
      * @param provider the provider name (google, facebook, etc.).
+     * @param withRefreshToken optional, include the refresh token in the response
+     * @param forceRefresh optional, force to refresh the token
      * @returns The ProviderTokenResponse of the given user and provider
      */
     getProviderToken: (
       loginId: string,
       provider: string,
+      withRefreshToken?: boolean,
+      forceRefresh?: boolean,
```

Comment:
I'd consider wrapping these in an "options object" so we can add future options without breakage

Reply:
done